lib/karma-oax.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+const execa = require('execa');
+const oaxPath = require('oax');
+const path = require('path');
+
+function createPreprocessor(config, logger) {
+	const log = logger.create('preprocessor.oax');
+
+	return async function preprocess(original, file, done) {
+		const originalPath = file.originalPath;
+		const location = path.relative(config.basePath, originalPath);
+
+		const oaxArguments = [];
+		oaxArguments.push(`--input ${file.originalPath}`);
+		if (!config.ignoreBower) {
+			oaxArguments.push(`--for-bower`);
+		}
+
+		const oaxCommand = oaxPath + ' ' + oaxArguments.join(' ');
+		try {
+			log.info('Generating bundle for ./%s', location);
+			const {stdout, stderr} = await execa.command(oaxCommand, {
+				cwd: config.basePath
+			});
```

Comment:
maybe we should add a runner to oax, so that the user doesn't have to build up the command themselves